cron-command.php

```diff
@@ -4,9 +4,9 @@
 	return;
 }
 
-$autoload = dirname( __FILE__ ) . '/vendor/autoload.php';
-if ( file_exists( $autoload ) ) {
-	require_once $autoload;
+$wpcli_cron_autoload = dirname( __FILE__ ) . '/vendor/autoload.php';
```

Comment:
We sticked with the convention of `$wpcli_<command>_autoloader` for now (note the "er" at the end).